content/rc/databases/view-edit-database.md

```diff
@@ -160,21 +150,27 @@ The **Databases** list summarizes the status of all databases from the subscript
 
 You can:
 
-- Sort the list in descending or ascending order using the the arrow displayed to right of the field name in the header.  Supported fields include **Subscription**, **Name**, and **Memory**.
+- Search by typing into the search box located above the database list.
```

Comment:
The style guide generally recommends avoiding directional language like "above x", but maybe it's okay to leave it in this case.